.github/workflows/build-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
```

Comment:
Why not drop 16.x while we're here?

Reply:
Since the version is still part of `node.engines. Changing that'd be a breaking and separate change. Feel free to submit a PR!